python/ray/scripts/scripts.py

```diff
@@ -759,90 +772,71 @@ def start(
                     cli_logger.print(
                         "the head node: ray start --head --node-ip-address=<public-ip>."
                     )
-                else:
-                    cli_logger.print(
-                        "Multi-node Ray clusters are not supported on OSX and Windows."
-                    )
-                    cli_logger.print(
-                        "If you would like to proceed anyway, restart Ray with:"
-                    )
-                    cli_logger.print(
-                        cf.bold("  ray stop"),
-                    )
-                    cli_logger.print(
-                        cf.bold("  {}=true ray start"),
-                        ray_constants.ENABLE_RAY_CLUSTERS_ENV_VAR,
-                    )
-                cli_logger.newline()
-            else:
-                cli_logger.print("To add another node to this Ray cluster, run")
-                # NOTE(kfstorm): Java driver rely on this line to get the address
-                # of the cluster. Please be careful when updating this line.
-                cli_logger.print(
-                    cf.bold("  ray start --address='{}'"),
-                    bootstrap_address,
-                )
-                cli_logger.newline()
-            if ray_constants.ENABLE_RAY_CLUSTER:
+
+            cli_logger.newline()
+            cli_logger.print(
+                "To connect to this Ray cluster, run `ray.init()` as usual:"
```

Comment:
```suggestion
                "To run your ray applications from a node of the Ray cluster, run `ray.init()` as usual:"
```
How about this? "Connect to" is a bit ambiguous to me.

Reply:
```
Connect to an existing Ray cluster or start one and connect to it.
```

This is the docstring of ray.init so I think we have been using this term all the time.